Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -398,6 +423,26 @@ - (void)trackHeartbeatEvents:(SEGTrackPayload *)payload
         return;
     }
 
+    if ([payload.event isEqualToString:@"Video Content Started"]) {
+        [self.ADBMediaHeartbeat trackPlay];
+        SEGLog(@"[ADBMediaHeartbeat trackPlay]");
+
+        self.mediaObject = [self createMediaObject:payload.properties andEventType:@"Content"];
+        [self.ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventChapterStart mediaObject:self.mediaObject data:payload.properties];
+        SEGLog(@"[ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventChapterStart mediaObject:%@ data:%@]", self.mediaObject, payload.properties);
+        return;
+    }
+
+    if ([payload.event isEqualToString:@"Video Content Completed"]) {
+        [self.ADBMediaHeartbeat trackComplete];
+        SEGLog(@"[ADBMediaHeartbeat trackComplete]");
+
+        self.mediaObject = [self createMediaObject:payload.properties andEventType:@"Content"];
+        [self.ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventChapterComplete mediaObject:self.mediaObject data:payload.properties];
```

Comment:
@brennan this differs from the [Android implementation here](https://github.com/segment-integrations/analytics-android-integration-adobe-analytics/blob/master/src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java#L565) I think we should consider also mapping a `trackEvent(MediaHeartbeat.Event.ChapterComplete, MediaObject, Map<String, String> metadata) ` in Android here — thoughts?

Reply:
Yes, I agree that we should add a `ChapterComplete` event - I can update this in the Android library. That said, it should probably be `heartbeat.trackEvent(MediaHeartbeat.Event.ChapterComplete, null, null);` - this is how Adobe implements `ChapterComplete` in their examples.

Reply:
Hmm yeah I see that [in iOS too](https://github.com/Adobe-Marketing-Cloud/video-heartbeat-v2/blob/master/sdks/iOS/samples/BasicPlayerSample/BasicPlayerSample/Classes/analytics/VideoAnalyticsProvider.m#L158).

I will change now